backend/Acebook.Test/TestDatabaseSetup.cs

```diff
@@ -35,8 +35,8 @@ private void SeedTestUser()
         conn.Open();
 
         using var cmd = new NpgsqlCommand(@"
```

Comment:
@RussellColes and @bookloversolmaz – looks like a test user is seeded into the DB here..? You can probably use this one in your tests